public/css/app.css

```diff
@@ -1,3725 +0,0 @@
-html {
```

Comment:
Added back in. Didn't know if you wanted compiled css committed. Thought I had added it in on accident so I ripped it out.